include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -130,6 +131,17 @@ typedef enum ONNXType {
   ONNX_TYPE_SPARSETENSOR,
 } ONNXType;
 
+// These types are synced with internal
+// SparseFormatFlags but are not exposed
+// as flags
+typedef enum OrtSparseFormat {
```

Comment:
Are there going to be no new APIs to allow users create a sparse tensor for now (i.e.) are sparse tensors limited to just sparse initializers in the model ?

Reply:
There are only python APIs in the PR as it is already too big. Much more is coming later.